README.md

```diff
@@ -6,6 +6,10 @@
 
 These samples provide a starting point for how to integrate different services using a Compose file and to manage their deployment with Docker Compose.
 
+> **Note:**
+>
+> The following samples use the build in development server to run your application. Do not use these in a production environment.
```

Comment:
I find the first sentence a little bit confusing. How about: 
`The following samples are intended for use in local development environments (project setups, tinkering with software stacks etc). The samples are not to be deployed in production environments.`

Reply:
Sounds good. Updated the text with minor changes